src/components/app-top-bar.js

```diff
@@ -204,6 +215,7 @@ const AppTopBar = ({ user, tabIndex, onChangeTab, userManager }) => {
     const studyUuid = useSelector((state) => state.studyUuid);
 
     const currentNode = useSelector((state) => state.currentTreeNode);
+    const disabled = !isNodeBuilt(currentNode);
```

Comment:
I don't like this new value for two reasons :
- "disabled" is too generic a name. It would mean that the topbar is disabled, when it is not (even if the current node is not built, the topbar's other buttons are still working)
- There is already disabled props in this file, and each time their values means something functionally, that's easy to understand.

I would remove this const and set the component's disabled props like this : 
`<RunButtonContainer
                            studyUuid={studyUuid}
                            currentNode={currentNode}
                            setIsComputationRunning={setIsComputationRunning}
                            disabled={!isNodeBuilt(currentNode) || isNodeReadOnly(currentNode)}
                        />`

Reply:
DOne